### PR TITLE
New leaderboard for hebrew - release Sunday

### PR DESCRIPTION
**Pull request by clefourrier**
Ready for light review

### CODE REVIEW

File: leaderboard-hebrew.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - *Source*: [A Translation of the Winograd Schema Challenge to Hebrew](https://www.cs.ubc.ca/~vshwartz/resources/winograd_he.jsonl), by Dr. Vered Schwartz.
+
+- **Translation**: This task assesses the model's proficiency in translating between English and Hebrew. It evaluates the linguistic accuracy, fluency, and the ability to preserve meaning across languages, highlighting the model’s capability in bilingual translation tasks.
+    - *Source*: [NeuLabs-TedTalks](https://opus.nlpl.eu/NeuLab-TedTalks/en&he/v1/NeuLab-TedTalks) aligned translation corpus.
+
+## Technical Setup
+
+The leaderboard is inspired by the [Open LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard), and uses the [Demo Leaderboard template](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard). Models that are submitted are deployed automatically using HuggingFace’s Inference Endpoints and evaluated through API requests managed by the lighteval library.
```
Comment (shaltielshmid) [inline leaderboard-hebrew.md:51]: Whoops! My bad - the link for the demo leaderboard template is wrong: It should be `https://huggingface.co/demo-leaderboard-backend`

Comment (shaltielshmid) [reply leaderboard-hebrew.md:51]: (Also, did you remove the lighteval link on purpose?)

Comment (clefourrier) [reply leaderboard-hebrew.md:51]: The ligtheval link was lost in the rephrase :sweat_smile:

Comment (clefourrier) [inline leaderboard-hebrew.md:51]:
```suggestion
The leaderboard is inspired by the [Open LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard), and uses the [Demo Leaderboard template](https://huggingface.co/demo-leaderboard-backend). Models that are submitted are deployed automatically using HuggingFace’s [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index) and evaluated through API requests managed by the [lighteval](https://github.com/huggingface/lighteval) library.
```

File: leaderboard-hebrew.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  org: HebArabNlpProject
+- user: OmerKo
+  guest: true
+  org: Webiks
+- user: clefourrier
+---
+
+# Introducing the Open Leaderboard for Hebrew LLMs!
+
+The field of language technology (specifically, LLMs) has seen an underrepresentation of resources dedicated to languages other than English.
```
Comment (shaltielshmid) [inline leaderboard-hebrew.md:19]: Can we please erase this line?

Comment (clefourrier) [inline leaderboard-hebrew.md:19]:
```suggestion
```

**Review by linoytsaban — APPROVED** [review]

**Pull request merged**